libtransmission/rpcimpl.cc

```diff
@@ -199,44 +199,36 @@ struct tr_rpc_idle_data
     tr_session* session = nullptr;
     tr_variant::Map args_out;
     tr_rpc_response_func callback;
-};
-
-void tr_rpc_idle_done_legacy(struct tr_rpc_idle_data* data, JsonRpc::Error::Code code, std::string_view result)
-{
-    // build the response
-    auto response_map = tr_variant::Map{ 3U };
-    response_map.try_emplace(TR_KEY_arguments, std::move(data->args_out));
-    response_map.try_emplace(TR_KEY_result, std::empty(result) ? JsonRpc::Error::to_string(code) : result);
-    if (auto& tag = data->id; tag.has_value())
-    {
-        response_map.try_emplace(TR_KEY_tag, std::move(tag));
-    }
 
-    // send the response back to the listener
-    data->callback(data->session, tr_variant{ std::move(response_map) });
-}
+    // TODO: api-compat
```

Comment:
Looks like this comment is outdated?

Reply:
Not a blocker for 7995.

I have a different rpcimpl PR almost ready and I can remove the line there.